01_prepare_host.sh

```diff
@@ -63,6 +63,18 @@ fi
 # shellcheck disable=SC1091
 source lib/releases.sh
 
+## Install krew
+if ! command -v kubectl krew 2>/dev/null; then
```

Comment:
This is true if `kubectl` is installed, regardless of whether krew is installed.

```
$ kubectl krew
Error: unknown command "krew" for "kubectl"
Run 'kubectl --help' for usage.
```

```
$ if command -v kubectl krew 2>/dev/null; then echo "krew is installed"; fi
/usr/bin/kubectl
krew is installed
```

Probably instead of using `command`, you want to test a `kubectl krew` command which has exit code 0 when krew is installed.